src/components/Tasks/Toolbar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+import * as EteSync from 'etesync';
+
+import Switch from '@material-ui/core/Switch';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import IconButton from '@material-ui/core/IconButton';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import QuickAdd from './QuickAdd';
+
+import { PimType } from '../../pim-types';
+
+interface PropsType {
+  defaultCollection: EteSync.CollectionInfo;
+  onItemSave: (item: PimType, journalUid: string, originalItem?: PimType) => Promise<void>;
+  showCompleted: boolean;
+  setShowCompleted: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+export default function Toolbar(props: PropsType) {
+  const { defaultCollection, onItemSave, showCompleted, setShowCompleted } = props;
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+      {defaultCollection && <QuickAdd onSubmit={onItemSave} defaultCollection={defaultCollection} />}
```

Comment:
I missed it in a previous change, but it looks like there's a hardcoded `25px` right margin in QuickAdd. It should be removed from there entirely (as this is not an internal property of QuickAdd) and instead added here.
The spacing is also wrong because:
1. It's too spacious.
2. It's using pixel sizing rather than `em`.
It should be changed to `0.75em` which I checked and looks good locally.